File: javascript/raindrops/raindrops.js
var Raindrops = function() {}

Raindrops.prototype.convert = function(num) {
    let returnValue = '';
    returnValue += num % 3 == 0 ? "Pling" : "";
    returnValue += num % 5 == 0 ? "Plang" : "";
    returnValue += num % 7 == 0 ? "Plong" : "";
    return returnValue == '' ? num.toString() : returnValue;
}

module.exports = Raindrops;

File: ruby/raindrops/raindrops.rb
class Raindrops
    def self.convert(num)
        str = ""
        str += "Pling" if num % 3 == 0
        str += "Plang" if num % 5 == 0
        str += "Plong" if num % 7 == 0
        return num.to_s if str == ""
        str
    end
end
module BookKeeping
    VERSION = 3
end